pennylane/tape/tapes/cv_param_shift.py

```diff
@@ -263,7 +275,14 @@ def processing_fn(results):
             shifted_forward = np.array(results[0])
             shifted_backward = np.array(results[1])
 
-            return c * (shifted_forward - shifted_backward)
+            stat = np.zeros_like(results[0])
+
+            for c, res in zip(coeffs, results):
+                shifted = np.array(res)
+                # stat += c * shifted
+                np.add(stat, c * shifted, out=stat, casting="unsafe")

```

Comment:
The workaround proposed by https://github.com/numpy/numpy/issues/7225#issuecomment-223218486.

Reply:
🤔 I'm guessing this is because `c` has a dtype that is causing casting issues?

Would it be better to simply cast/ensure that `c` and `shifted` are the correct dtype?